package.json

```diff
@@ -17,7 +17,7 @@
   },
   "homepage": "https://github.com/integr8ly/tutorial-web-app-walkthroughs#readme",
   "devDependencies": {
-    "integreatly-asciidoc": "0.0.2"
+    "integreatly-asciidoc": "0.0.4"
```

Comment:
@aidenkeating  what's this?

Reply:
@finp `integreatly-asciidoc` is a library for validating that walkthroughs have all the required files and sections in those files, like the JSON config structure etc.

Reply:
gotcha, I just realised it's in devDependencies of package.json, thought I was looking at walkthrough.json